EventBrite/Fields/EventBriteField.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Etch.OrchardCore.Fields.EventBrite.Models;
+using OrchardCore.ContentManagement;
+using System.Collections.Generic;
+
+namespace Etch.OrchardCore.Fields.EventBrite.Fields
+{
+    public class EventBriteField : ContentField
+    {
+        public string Value { get; set; }
+        public EventBriteItem EventBriteData { get; set; }
```

Comment:
I'd change this property to `Data`. It's kind of implied it's "EventBrite" related. Remove the clutter.